solr/webapp/web/partials/query.html

```diff
@@ -23,6 +23,18 @@
       </label>
       <input type="text" ng-model="qt" id="qt" placeholder="/select" value="/select" title="Request handler in solrconfig.xml.">
 
+      <div>
```

Comment:
I would recommend  that field should be optional. If the core has paramsets then it should be shown. otherwise it should not be shown..
But it's up to you